pkg/api/v1/control-plane/types/types.go

```diff
@@ -0,0 +1,43 @@
+package types
```

Comment:
I think we should treat this as a breaking change and therefore should have a higher minor version for changelog

Reply:
circular dependency in `pkg/api/v1/control-plane/cache/status.go`